backend/src/packages/meditation/libs/types/meditation-create-query-payload.ts

```diff
@@ -0,0 +1,14 @@
+import { type ContentType } from '#libs/enums/enums.js';
+import { type ValueOf } from '#libs/types/types.js';
+
+type MeditationCreateQueryPayload = {
+  id: number;
+  mediaUrl: string;
+  contentType: ValueOf<typeof ContentType>;
+  topic: {
+    id?: number;
+    name: string;
+  };
```

Comment:
topics will be created separately
```suggestion
  topicId
```

Reply:
@Liza-Veis What sould I do at now if there's no entered topic in `meditation_topics` table?

Reply:
I made it null in this case, so done